src/components/cell/CellDivider.svelte

```diff
@@ -49,4 +43,8 @@
     @apply flex items-center bg-white z-10 px-2.5 py-1 text-xs rounded-md shadow
       transition-colors hover:bg-gray-50 active:bg-gray-200;
   }
+
+  .divider > button:not(:first-child) {
+    @apply ml-4;
+  }
```

Comment:
Minor but you might be able to just apply `space-x-4` to the container. :)

Reply:
Nice catch.